proto/autokitteh/sessions/v1/svc.proto

```diff
@@ -74,6 +74,13 @@ message GetResponse {
 
 message GetLogRequest {
   string session_id = 1 [(buf.validate.field).string.min_len = 1];
+
+  // If set, all values returned will be string values
+  // that contained the native values in JSON format.
```

Comment:
Same as in `proto/autokitteh/events/v1/svc.proto`

Reply:
refed